Radzen.Blazor/RadzenDatePicker.razor

```diff
@@ -96,6 +96,9 @@
                                 </tbody>
                             </table>
                         </div>
+                        <div class="rz-datepicker-footer">
+                             @FooterTemplate
```

Comment:
It will be good to check if the FooterTemplate is defined before rendering this element

Reply:
Done